qiskit/tools/monitor/job_monitor.py

```diff
@@ -19,20 +19,25 @@
     from IPython.display import display    # pylint: disable=import-error
 
 
-def _text_checker(job, interval, _interval_set=False):
+def _text_checker(job, interval, _interval_set=False, quiet=False, to_err=False):
```

Comment:
Instead of to_err and a boolean flag here, what do you think about doing an `output` kwarg that takes in any file like object. We can default it to sys.stdout but if a user wants stderr they can pass in sys.stderr. But, then they can also pass in an io stream an actual file object, etc.

Reply:
Ha, that's actually what I do in my own code. Sought simplicity here.
I can make that change.